pages/login.js

```diff
@@ -1,93 +1,68 @@
-/* eslint-disable no-undef */
 import { useForm } from 'react-hook-form'
-import Image from 'next/image'
-import Head from 'next/head'
 import { providers, signIn, getSession } from 'next-auth/client'
 import { useRouter } from 'next/router'
-import Navbar from '../components/layout/Navbar'
 
 export default function Login () {
   const { register, handleSubmit, formState: { errors } } = useForm()
   const router = useRouter()
-  const loginUser = async (data) => {
-    const result = await signIn('credentials', {
-      redirect: false,
-      email: data.email,
-      password: data.password
-    })
 
-    if (result.error !== null) {
-      if (result.status === 401) {
-        resultError = 'Your username/password combination was incorrect. Please try again'
-      } else {
-        resultError = result.error
-      }
-    } else {
+  const loginUser = async (data) => {
+    try {
+      // eslint-disable-next-line no-unused-vars
```

Comment:
no need for disabling eslint here (generally it's a bad practice to disable es-lint, better to search for a solution without needing to disable eslint)

can be done without variable assignment:
```
await signIn('credentials', {
        redirect: false,
        email: data.email,
        password: data.password
      })
```

Reply:
done